main.go

```diff
@@ -833,6 +833,13 @@ func initialiseSystem() error {
 		if globalConf.PIDFileLocation == "" {
 			globalConf.PIDFileLocation = "/var/run/tyk/tyk-gateway.pid"
 		}
+		if globalConf.UseDBAppConfigs && globalConf.Policies.PolicySource != "service" {
+			globalConf.Policies.PolicySource = "service"
+			globalConf.Policies.PolicySource = globalConf.DBAppConfOptions.ConnectionString
```

Comment:
looks like it is supposed to be `globalConf.Policies.PolicyConnectionString `

Reply:
good spot